atc/metric/emitter/prometheus.go

```diff
@@ -305,6 +306,17 @@ func (config *PrometheusConfig) NewEmitter() (metric.Emitter, error) {
 	)
 	prometheus.MustRegister(resourceChecksVec)
 
+	checksVec := prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Namespace: "concourse",
+			Subsystem: "lidar",
+			Name:      "checks_total",
+			Help:      "Counts the number of checks finished",
+		},
+		[]string{"scopeID", "checkName"},
```

Comment:
heey, do you think we really need these two?

Assuming that some of these timeseries dbs (like prometheus' tsdb) create a timeseries for each labelset (combination of those labels like `scopeID` and `checkName`), this can get super expensive without really bringing much value.

wdyt of  just dropping those two? 🤔  would an operator lose much? IMO it's worth dropping - we aggregate that with a `sum` in our dashboards anyway 😅  

thx!

Reply:
(if we go without those two dimensions, we'd then go from `NewCounterVec` to `NewCounter`, and then in the place where we do the increments, we'd go for a simple `Inc()` instead of  chaining`WithLabelValues()`)